daffodil-japi/src/main/scala/org/apache/daffodil/japi/Daffodil.scala

```diff
@@ -17,47 +17,48 @@
 
 package org.apache.daffodil.japi
 
-import org.apache.daffodil.compiler.{ Compiler => SCompiler }
+import org.apache.daffodil.compiler.{Compiler => SCompiler}
```

Comment:
I don't know which style is closer to the standard Scala formatting as implemented by the unmodified defaults of the Scalariform tool, but let's make any formatting changes in a separate PR that reformats the entire codebase uniformly.  Will you please remove these formatting changes which your IDE must have made? 

Also, which IDE are you using?  I checked two pages in our Wiki (https://cwiki.apache.org/confluence/display/DAFFODIL/Code+Style+Guidelines and https://cwiki.apache.org/confluence/display/DAFFODIL/IntelliJ+IDEA+Setup) and found nothing there which tells people how to fix this particular formatting default in IntelliJ IDEA.  If you're using IDEA, that's an oversight which we need to fix - please let me know how to find the default after you change it and I'll update the Wiki for you.

Reply:
Yeah, unintentional, repaired.

> Also, which IDE are you using? 

Intellij.  I use [Scalafmt](https://scalameta.org/scalafmt/) for nearly everything else, never adjusted for this project.

> ... how to fix this particular formatting default in IntelliJ IDEA

@mbeckerle's [comment here](https://cwiki.apache.org/confluence/display/DAFFODIL/IntelliJ+IDEA+Setup?focusedCommentId=148644684#comment-148644684) fixes it.